<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinaryThresholdFilterDialog</class>
 <widget class="QDialog" name="BinaryThresholdFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Binary threshold filter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Binary threshold filter:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Inside value:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxInsideValue">
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSlider" name="horizontalSliderInsideValue">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Outside value:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxOutsideValue">
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSlider" name="horizontalSliderOutsideValue">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Lower threshold:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxLowerThreshold">
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSlider" name="horizontalSliderLowerThreshold">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Upper threshold</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBoxUpperThreshold">
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSlider" name="horizontalSliderUpperThreshold">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxInsideValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderInsideValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxOutsideValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderOutsideValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxLowerThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderLowerThreshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxUpperThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderUpperThreshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderInsideValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxInsideValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderOutsideValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxOutsideValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderLowerThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxLowerThreshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderUpperThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxUpperThreshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>BinaryThresholdFilterDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
